app/src/docs/v1.api-spec.yaml

```diff
@@ -17,30 +17,348 @@ security:
+        '200':
+          description: Returns the object information
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ObjectDetails'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '422':
+          $ref: '#/components/responses/UnprocessableEntity'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+    post:
+      summary: Create an Object
+      description: >-
+        This endpoint will create an object.
+      operationId: PostCreateObject
+      tags:
+        - Object
+      responses:
+        '201':
+          description: Returns create Object response.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MergeResponse'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '409':
+          $ref: '#/components/responses/Conflict'
+        '422':
+          $ref: '#/components/responses/UnprocessableEntity'
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+  '/{objId}':
+    get:
+      summary: Details of a single Object
+      description: >-
+        This endpoint returns all of the information related to
+        objId.
+      operationId: GetObject
       tags:
-        - Hello
+        - Object
+      parameters:
+        - $ref: '#/components/parameters/PathMessageId'
       responses:
         '200':
-          description: Hello result
+          description: Returns the object information
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/HelloDetail'
+                $ref: '#/components/schemas/ObjectDetails'
         '401':
           $ref: '#/components/responses/UnauthorizedError'
         '403':
           $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '422':
+          $ref: '#/components/responses/UnprocessableEntity'
         default:
           description: Unexpected error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+    post:
+      summary: Updates an Object
+      description: >-
+        This endpoint will update an object.
+      operationId: PostUpdateObject
+      tags:
+        - Object
+      parameters:
+        - $ref: '#/components/parameters/PathMessageId'
```

Comment:
you'll have to rename 'PathObjectId' i think some of these schema names are specific to CHES concepts

Reply:
Renaming all instances of this to PathObjectId